chainer/functions/__init__.py

```diff
@@ -63,6 +63,8 @@
 from chainer.functions.array.hstack import hstack  # NOQA
 from chainer.functions.array.im2col import im2col  # NOQA
 from chainer.functions.array.im2col import Im2Col  # NOQA
+from chainer.functions.array.moveaxis import moveaxis  # NOQA
+from chainer.functions.array.moveaxis import Moveaxis  # NOQA
```

Comment:
Remove the line since we have decided not to expose `FunctionNode` implementation.